### PR TITLE
Add example for proxy.istio.io/config annotation

### PR DESCRIPTION
**Pull request by jacob-delgado**
Add example to tracing section.

Do not merge until 1.6.1

### CODE REVIEW

File: content/en/docs/tasks/observability/distributed-tracing/configurability/index.md [hunk 1]
```diff
@@ -27,6 +27,35 @@ EOF
 
 Then, you can append any configuration options to the `tracing.yaml` file.
 
+## Add `istio.proxy.io/config` annotation to your Pod metadata specification
```
Comment (douglas-reid) [inline content/en/docs/tasks/observability/distributed-tracing/configurability/index.md:30]: (here and elsewhere): shouldn't this be `proxy.istio.io/config` ?

Comment (jacob-delgado) [reply content/en/docs/tasks/observability/distributed-tracing/configurability/index.md]: addressed

File: content/en/docs/tasks/observability/distributed-tracing/configurability/index.md [hunk 2]
```diff
@@ -27,6 +27,35 @@ EOF
 
 Then, you can append any configuration options to the `tracing.yaml` file.
 
+## Add `proxy.istio.io/config` annotation to your Pod metadata specification
+
+There are occasions where you may wish to override the mesh-wide setting for a Pod-specific
```
Comment (frankbu) [inline content/en/docs/tasks/observability/distributed-tracing/configurability/index.md:32]:
```suggestion
There are occasions where you may wish to override the mesh-wide configuration for a Pod-specific
```

File: content/en/docs/tasks/observability/distributed-tracing/configurability/index.md [hunk 3]
```diff
@@ -27,6 +27,35 @@ EOF
...
+    metadata:
+      ...
+      proxy.istio.io/config: |
+        tracing:
+          ...                            # custom tracing definition
+    spec:
+      ...
+{{< /text >}}
+
+Then, you can append any configuration specified as mentioned below to change tracing settings
```
Comment (frankbu) [inline content/en/docs/tasks/observability/distributed-tracing/configurability/index.md:56]:
```suggestion
You can then append any of the settings specified below, to change the tracing configuration
```

**Pull request merged**